### PR TITLE
Increase the length of varlist to hold more variables for recentering

### PR DESCRIPTION
**Pull request by chunhuazhou**
## DESCRIPTION OF CHANGES:

Increase the length of varlist to hold more variables for recentering. 

## TESTS CONDUCTED:

Tested on Jet

### CODE REVIEW

File: ens_mean_recenter/ens_mean_recenter.f90 [hunk 1]
```diff
@@ -43,11 +43,12 @@ PROGRAM ens_mean_recenter
 ! namelist
   integer, parameter    :: max_num_file = 10
   integer, parameter    :: filename_len=200
+  integer, parameter    :: varlist_len=500
   integer :: fv3_io_layout_y
   character(len=filename_len)  :: filebase
   character(len=filename_len)  :: filetail(max_num_file)
   integer                      :: numvar(max_num_file)
-  character(len=filename_len)  :: varlist(max_num_file)
+  character(len=varlist_len)  :: varlist(max_num_file)
```
Comment (guoqing-noaa) [inline ens_mean_recenter/ens_mean_recenter.f90]: It looks like you don't need a new variable.
may use 500 directly just like what is done in `module_fv3lam_stats.f90`, :)

Comment (hu5970) [reply ens_mean_recenter/ens_mean_recenter.f90]: either way works for me.

Comment (chunhuazhou) [reply ens_mean_recenter/ens_mean_recenter.f90]: Added this new variable to be consistent with the rest of the code. But definitely can use 500 directly.

**Review by guoqing-noaa — APPROVED** [review]
Thanks for the change. It is not an import issue actually. 
A better way may be to define 500 as a variable somewhere and both f90 files can refer the same variable.
However, it only involve two files, so explicitly setting 500 looks fine to me.

**Review by hu5970 — APPROVED** [review]

**Pull request merged**